cluster/images/etcd/Makefile

```diff
@@ -40,13 +40,19 @@ REVISION?=0
 IMAGE_TAG=$(LATEST_ETCD_VERSION)-$(REVISION)
 
 ARCH?=amd64
+ALL_ARCH = amd64 arm arm64 ppc64le s390x
```

Comment:
Do we need to bump REVISION?

Reply:
do we really need to update the revision? @ixdy

Reply:
we already have `3.2.24-0` in registry already @mkumatag ( https://console.cloud.google.com/gcr/images/google_containers/GLOBAL/etcd?gcrImageListsize=50 )

Reply:
`3.2.24 != 3.2.24-0`

Reply:
There's also a `3.2.24` already.

This might be problematic, since I don't think our image promotion process allows overwriting an existing tag.

Reply:
tags are mutable ;-) TBH this is googles release jiggery, so I don't have strong opinions wrt -revision.

Reply:
Safest approach would be to bump REVISION to 1, publish 3.2.24-1, and overwrite 3.2.24 to also be the same image as 3.2.24-1.

We'll eventually deprecate the tags that don't have the revision suffix, but we're keeping it around for backward compatibility for now.

Reply:
I also like @jpbetz's idea ^. What do others think?

Reply:
i think i'm OK with revisions as long as everything is manifest list. :)